core/src/global/test/config-controller.spec.ts

```diff
@@ -18,14 +19,14 @@ describe('Config', () => {
       bool4: 'hola',
       bool5: 0,
       bool6: 1,
-    });
-    expect(config.getBoolean('bool0')).toEqual(false);
-    expect(config.getBoolean('bool1')).toEqual(false);
-    expect(config.getBoolean('bool2')).toEqual(true);
-    expect(config.getBoolean('bool3')).toEqual(true);
-    expect(config.getBoolean('bool4')).toEqual(false);
-    expect(config.getBoolean('bool5')).toEqual(false);
-    expect(config.getBoolean('bool6')).toEqual(true);
+    } as any);
+    expect(config.getBoolean('bool0' as any)).toEqual(false);
```

Comment:
Should these be typed as `keyof IonicConfig` instead?

```suggestion
    expect(config.getBoolean('bool0' as keyof IonicConfig)).toEqual(false);
```

Reply:
`bool0` is not a key of that interface. So we would either need to do `as unknown as keyof IonicConfig` or cast to `any` here. Since we aren't using the key anywhere else, I opted for `any`. 

These legacy tests should probably get an overhaul in the future to use valid keys or provide an interface shape for the config that matches what the test is accessing.

Reply:
Oh okay. When I tested it out locally it worked for me but maybe I needed to refresh VsCode or something.